packages/api/tests/endpoints/test-reconciliation-reports.js

```diff
@@ -293,22 +296,28 @@ test.serial('create a report starts an async operation', async (t) => {
       id,
     });
 
-    t.true(stub.calledWith({
-      asyncOperationTaskDefinition: process.env.AsyncOperationTaskDefinition,
-      cluster: process.env.EcsCluster,
-      lambdaName: process.env.invokeReconcileLambda,
-      description: 'Create Reconciliation Report',
-      operationType: 'Reconciliation Report',
-      payload: normalizeEvent({}),
-      useLambdaEnvironmentVariables: true,
-    }));
+    t.deepEqual(
+      omit(stub.lastCall.args[0], ['knexConfig']),
```

Comment:
Hmm.     Well in this case, it was explicitly omitting the knexConfig, but I'm not sure I recall *why* that mattered.  This may be a victim of refactoring, but I agree it doesn't appear to be needed given the current state. 

216133a18 seems to be fine.